config/helpers.go

```diff
@@ -11,12 +11,21 @@ const (
 	ServiceAuthenticate = "authenticate"
 	// ServiceCache represents running the cache service component
 	ServiceCache = "cache"
+	// ServiceForwardAuth represents running the forward auth service component
+	ServiceForwardAuth = "forwardauth"
 	// StorageRedisName is the name of the redis storage backend
 	StorageRedisName = "redis"
 	// StorageInMemoryName is the name of the in-memory storage backend
 	StorageInMemoryName = "memory"
+	// ProxyTypeNginx is the name of nginx proxy.
+	ProxyTypeNginx = "nginx"
+	// ProxyTypeTraefik is the name of traefik proxy.
+	ProxyTypeTraefik = "traefik"
 )
 
+// ProxyTypes contains all supported proxy types.
```

Comment:
To avoid confusing in a proxy-product where we use the term Proxy generically all over the place... maybe we should call this (and it's members) something like `ForwardingProxyTypes` and `ForwardingProxyNginx` and `ForwardingProxyTraefik`... thoughts?

Reply:
👌